packages/sources/layer2-sequencer-health/src/config/index.ts

```diff
@@ -64,5 +84,15 @@ export const makeConfig = (prefix?: string): ExtendedConfig => {
   const deltaBlocks = Number(util.getEnv('DELTA_BLOCKS', prefix)) || DEFAULT_DELTA_BLOCKS
   const timeoutLimit = Number(util.getEnv('NETWORK_TIMEOUT_LIMIT', prefix)) || DEFAULT_TIMEOUT_LIMIT
   config.defaultEndpoint = DEFAULT_ENDPOINT
-  return { ...config, delta, deltaBlocks, timeoutLimit }
+  const starkwareConfig: ExtendedConfig['starkwareConfig'] = {
```

Comment:
We should only append this config if we're running a starknet instance of the EA.

Reply:
So every instance of this EA typically supports fetching the health of all networks.  The only way we'd be able to differentiate between EAs that run Starknet vs those that don't is to set another env var.  I think we shouldn't do this though as it adds an unnecessary env var.

Reply:
You're right!